src/dbscan.py

```diff
@@ -1,10 +1,24 @@
+# pylint: disable=invalid-name
```

Comment:
because the linter doesn't want caps for variable names but for me it's clearer to put caps on vectors names (`X` instead of `x`)  when they have more than 1 dimension and not be confused with the first coordinate of two dimensional vector `(x,y)`

Reply:
I don't mind changing it but it feels more aligned with the "ML convention" of `Y = model(X)`

Reply:
that's ok then, you can keep the majuscule variable